### PR TITLE
Fix time-related tests in datanommer.commands.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg.d/datanommer.py [hunk 1]
```diff
@@ -15,5 +15,5 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 config = {
     'datanommer.enabled': True,
-    'datanommer.sqlalchemy.url': 'sqlite:///datanommer.db',
+    'datanommer.sqlalchemy.url': 'postgres://postgres:alksjdf@localhost/datanommer',
```
Comment (pypingou) [inline fedmsg.d/datanommer.py]: Wrong commit :)

File: datanommer.commands/tests/test_commands.py [hunk 2]
```diff
@@ -657,9 +664,9 @@ def test_latest_timesince(self):
             gc.return_value = self.config
 
             now = datetime.now()
-            time1 = now.replace(day=now.day-1)
-            time2 = now.replace(minute=now.minute-1)
-            time3 = now.replace(second=now.second-1)
+            time1 = now - timedelta(days=1)
+            time2 = now - timedelta(seconds=60)
+            time3 = now - timedelta(seconds=1)
```
Comment (pypingou) [inline datanommer.commands/tests/test_commands.py:669]: This part surprises me a little

**Comment by pypingou** [comment]
Except for one change (last file), and one small question :+1:

File: datanommer.commands/tests/test_commands.py (hunk 2 above)
Comment (ralphbean) [inline datanommer.commands/tests/test_commands.py:669]: Yeah.  Try setting the day to `0`.  :boom:

**Pull request merged**